### PR TITLE
kubecon na 2019 roundup blog post

### PR DESCRIPTION
**Pull request by wmorgan**
Signed-off-by: William Morgan <william@buoyant.io>

### CODE REVIEW

File: linkerd.io/content/blog/kubecon-na-2019-linkerd-roundup.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+draft: false
+slug: linkerd-at-kubecon-na-2019-roundup
+tags: [Community, Linkerd, Kubecon]
+thumbnail: /uploads/kccnceu19-linkerd.jpg
+---
+
+![Linkerd at KCCNCNA19](/uploads/kccncna19-linkerd.jpg)
+
+Once again, Linkerd momentum was at an all-time high at last week's KubeCon NA
+in San Diego. This was the biggest Kubecon yet, with 10k+ attendees, hundreds
```
Comment (alpeb) [inline linkerd.io/content/blog/kubecon-na-2019-linkerd-roundup.md]: Kubecon was capitalized differently in these two sentences.

File: linkerd.io/content/blog/kubecon-na-2019-linkerd-roundup.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+steam](https://searchitoperations.techtarget.com/news/252474376/Linkerd-vs-Istio-battle-heats-up-as-service-mesh-gains-steam):
+
+"We performance tested Istio and Linkerd just over a year ago, and chose
+Linkerd," said Cody Vandermyn, senior engineer at Nordstrom, a large retailer
+headquartered in Seattle. "At the time, our tests found that for us, Linkerd
+introduced less latency and met our requirements around memory footprint."
+
+## Wrapping it all up
+
+This was another great Kubecon for Linkerd. As usual, the best part for me is
```
Comment (ihcsim) [inline linkerd.io/content/blog/kubecon-na-2019-linkerd-roundup.md]: s/Kubecon/KubeCon

File: linkerd.io/content/blog/kubecon-na-2019-linkerd-roundup.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+Linkerd is a community project and is hosted by the [Cloud Native Computing
+Foundation](https://cncf.io). If you have feature requests, questions, or
+comments, we'd love to have you join our rapidly-growing community! Linkerd is
+hosted on [GitHub](https://github.com/linkerd/), and we have a thriving
+community on [Slack](https://slack.linkerd.io),
+[Twitter](https://twitter.com/linkerd), and the [mailing
+lists](https://linkerd.io/2/get-involved/). Come and join the fun!
+
+[vscode]: https://marketplace.visualstudio.com/items?itemName=bhargav.vscode-linkerd
```
Comment (ihcsim) [inline linkerd.io/content/blog/kubecon-na-2019-linkerd-roundup.md]: Probably not needed; looks like a copy-and-paste remnant from the KubeCon EU blog.

File: linkerd.io/content/blog/kubecon-na-2019-linkerd-roundup.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+{{< youtube sD7hCwq3Gw0 >}}
+
+## Microsoft, Linkerd, CI/CD, and OPA
+
+As usual, lots of great Linkerd content from our friends at Microsoft. Brian
+Redmond gave some great Linkerd demos in a talk entitled [Supercharge Your
+Microservices CI/CD with Service Mesh and
+Kubernetes](https://www.youtube.com/watch?v=SMoaem3UBag), and Microsoft's Rita
+Zhang joined Linkerd maintainer Ivan Sim for a presentation on [Enforcing
```
Comment (scottcarol) [inline linkerd.io/content/blog/kubecon-na-2019-linkerd-roundup.md]: this is a nit, but you said Ivan is a Linkerd maintainer and Charles is a Linkerd engineer above, do you want these to be the same titles?

File: linkerd.io/content/blog/kubecon-na-2019-linkerd-roundup.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+[Linkerd Deep Dive](https://www.youtube.com/watch?v=NqjRqe0J98U) talk. Both
+talks were jam-packed with people and had some really great Q&A.
+
+{{< youtube guHZ7U7ZoWc >}}
+{{< youtube NqjRqe0J98U >}}
+
+## Linkerd in the news
+
+Linkerd's momentum is not lost on the press. I particularly like this quote
+from Cody in entitled [Linkerd vs. Istio battle heats up as service mesh gains
```
Comment (scottcarol) [inline linkerd.io/content/blog/kubecon-na-2019-linkerd-roundup.md]: remove `entitled`

File: linkerd.io/content/blog/kubecon-na-2019-linkerd-roundup.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+
+## Microsoft, Linkerd, CI/CD, and OPA
+
+As usual, lots of great Linkerd content from our friends at Microsoft. Brian
+Redmond gave some great Linkerd demos in a talk entitled [Supercharge Your
+Microservices CI/CD with Service Mesh and
+Kubernetes](https://www.youtube.com/watch?v=SMoaem3UBag), and Microsoft's Rita
+Zhang joined Linkerd maintainer Ivan Sim for a presentation on [Enforcing
+Automatic mTLS With Linkerd and OPA
+Gatekeeper](https://www.youtube.com/watch?v=gMaGVHnvNfs)
```
Comment (scottcarol) [inline linkerd.io/content/blog/kubecon-na-2019-linkerd-roundup.md]: do you want a period at the end of the talk title?

File: linkerd.io/content/blog/kubecon-na-2019-linkerd-roundup.md [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+and Cody Vendermyn titled [Service Mesh: There and Back
+Again](https://www.youtube.com/watch?v=sq8nsjuJqO4), in which they detail
+Nordstrom's service mesh journey and why they chose Linkerd.
+
+{{< youtube sq8nsjuJqO4 >}}
+
+## Serverless + Service Mesh with Linkerd and OpenFaaS
+
+OpenFaaS founder Alex Ellis joined Linkerd engineer Charles Pretzer for a great
+talk entitled [OpenFaaS Cloud plus Linkerd colon A Secure, Multi-Tenant
```
[the remark this answers is not in the document]
Comment (wmorgan) [reply linkerd.io/content/blog/kubecon-na-2019-linkerd-roundup.md]: lol

**Comment by wmorgan** [comment]
Thanks friends!

**Pull request merged**